services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -195,3 +288,116 @@ def produce_map_01_credential_payload(cls, did: str, permit_amendment: PermitAme
             }
         }
         return credential
+
+    @classmethod
+    def produce_untp_cc_map_payload(cls, did: str, permit_amendment: PermitAmendment):
+        """Produce payload for Mines Act Permit UNTP Conformity Credential from permit amendment and did."""
+        ANONCRED_SCHEME = "https://hyperledger.github.io/anoncreds-spec/"
+
+        curr_appt = permit_amendment.permittee_appointments[0]
+        for pa in permit_amendment.permittee_appointments:
```

Comment:
Any reason to do this comparison again? Just seeing as `order_by start_date desc` was already added to the new  permittee_appointments relationship

Reply:
That was so bad that was confusing and wrong at the same time. See change and comment.